JMMServer/Providers/MyAnimeList/MALHelper.cs

```diff
@@ -530,15 +532,21 @@ public static void UpdateMALSeries(AnimeSeries ser)
                             ser.GetAnime().AnimeID);
                         continue;
                     }
+
+                    string confirmationMessage = "";
+                    if (animeListHashtable.ContainsKey(malID))
```

Comment:
I think you were making the right choice, but it would seem that some time in the past couple of months the API responses changed :(